src/frameworks/next.json

```diff
@@ -14,5 +14,11 @@
     "command": "next build",
     "directory": "out"
   },
-  "env": {}
+  "env": {},
+  "plugins": [
+    {
+      "packageName": "@netlify/plugin-nextjs",
+      "schema": { "min_node_version": "10.13.0" }
```

Comment:
I like this declarative approach! :heart: 

Should `schema` be named according to what the property intent is (as opposed to its format, JSON schema), for example `condition`?

Reply:
Good point! 
Done in https://github.com/netlify/framework-info/pull/111/commits/4518b30cef2585279700f551c8a50ecfc041a62f